### ISSUE TITLE
Incorrect behavior when generating "args" interface if the argument is an enumeration

### ISSUE THREAD

**Issue by OniVe** [prisma/graphql-resolver-codegen#53]
**Bug description**

Error in file `./generated/resolver.ts`: Cannot find name 'T'

```typescript
export namespace QueryResolvers {
  export interface ArgsUsers {
    orderBy: T["UsersOrderByInput"] | null;
  }
}
```

**To Reproduce**

Add new test `enum-import.graphql`:

```gql
enum UsersOrderByInput {
  name_ASC
  name_DESC
}

type Query {
  users(orderBy: UsersOrderByInput): [User!]!
}
```

**Expected behavior**

`./generated/resolver.ts`
```typescript

enum UsersOrderByInput {
  "name_ASC",
  "name_DESC"
}
// ...
export namespace QueryResolvers {
  export interface ArgsUsers {
    orderBy: UsersOrderByInput | null;
  }
}
```
**Additional context**
A similar error occurs for related fields:

`./src/schema.graphql`
```gql
enum MessagesOrderByInput {
  createdAt_ASC
  createdAt_DESC
}
# ...
type User {
  id: ID!
  # ...
  receivedMessages(orderBy: MessagesOrderByInput): [Message!]!
}
```

`./generated/resolver.ts`
```typescript

export namespace UserResolvers {
  export interface ArgsReceivedMessages {
    orderBy: T["MessageOrderByInput"] | null;
  }
}
```

**Versions:**
 - graphql-resolver-codegen - 0.2.4

**Comment by tom8192** [comment]
I have exactly the same problem, what's the temporary solution until the package is fixed?

**Comment by divyenduz** [comment]
This is resolved via https://github.com/prisma/graphql-resolver-codegen/pull/68

Released in `0.3.1@beta` - `npm install -g graphql-resolver-codegen@beta`
